core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3008,7 +3107,8 @@ class KafkaApis(val requestChannel: RequestChannel,
       })
     }
 
-    if (!authHelper.authorize(request.context, ALTER, CLUSTER, CLUSTER_NAME)) {
+    // not yet supported when using a Raft-based metadata quorum
+    if (usingRaftMetadataQuorum || !authHelper.authorize(request.context, ALTER, CLUSTER, CLUSTER_NAME)) {
```

Comment:
`CLUSTER_AUTHORIZATION_FAILED` seems like a surprising error for an unsupported operation. Any of the following errors might be better:
-  `UNKNOWN_SERVER_ERROR` 
- `OPERATION_NOT_ATTEMPTED`
- `UNSUPPORTE_VERSION`

Reply:
Changed to `UnsupportedVersionException`